cli/src/deno.rs

```diff
@@ -11,29 +11,26 @@ use deno_core::{
     ModuleType, RuntimeOptions,
 };
 
+use crate::commands::extensions::{api_decls, ExtensionState};
+
 /// Deno runtime state.
 pub struct DenoRuntime {
     runtime: JsRuntime,
 }
 
-impl Default for DenoRuntime {
-    fn default() -> Self {
-        Self::new()
-    }
-}
-
 impl DenoRuntime {
     /// Create a new Deno runtime.
-    pub fn new() -> Self {
-        // TODO: Add Phylum API methods here.
-        let phylum_api = Extension::builder().build();
+    pub fn new(deps: ExtensionState) -> Self {
```

Comment:
I don't think `deps` is a great variable name here? It says basically nothing about what it is or how it is used. I'd either name it something like shared state, or phylum api.